region/maridia/inner-green.json

```diff
@@ -143,6 +143,16 @@
               "id": 4,
               "requires": [
                 {"or": [
+                  {"canComeInCharged": {
```

Comment:
Pretty cool that it gets you directly to the top ledge. Should these sparks require either gravity or suitlessMaridia?

Reply:
I opted no since it's such a miniscule amount of time that's spent doing Suitless things.